clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/TransferMetadata.scala

```diff
@@ -0,0 +1,39 @@
+package org.broadinstitute.clio.transfer.model
+
+/**
+  * Common information for metadata in all clio indexes.
+  */
+trait TransferMetadata[M <: TransferMetadata[M]] { self: M =>
+
+  /**
+    * Notes used to track changes to metadata (primarily deletions).
+    */
+  val notes: Option[String]
+
+  /**
+    * Paths to all files that should be moved by the clio-client move
+    * command for this metadata's index.
+    */
+  def pathsToMove: Seq[String]
+
+  /**
+    * Rebase all paths in `pathsToMove` onto a destination directory path.
```

Comment:
Not sure if `rebase` is the best term here. Also it's unclear what this does; I assume it creates a new model but it sounds like it modifies the current object.